agr_java_core/src/main/java/org/alliancegenome/cache/repository/helper/JsonResultResponse.java

```diff
@@ -101,6 +102,7 @@ public void addDistinctFieldValueSupplementalData(Map object) {
 		supplementalData.put(DISTINCT_FIELD_VALUES, object);
 	}
 
+	@JsonIgnore
```

Comment:
Isn't this used all around the site? Why would we need a @JsonIgnore?

Reply:
I wonder if anyone is using this. The distinctFieldValues should only used as part of the supplementalData, otherwise it is duplicated in the returned apyload.

Reply:
The UI solely accesses this object from the supplementalData object and makes this top-level output superfluous.

Reply:
Adam pointed out that there is a util function that retrieves the distinctFieldValues:
```

export const getDistinctFieldValue = (response, field) => {
  response = response || {};
  const {distinctFieldValues = {}} = response.supplementalData || {};
  return (distinctFieldValues[field] || [])
    .sort(compareAlphabeticalCaseInsensitive)
     // TODO: remove when backend is fixed, see https://agr-jira.atlassian.net/browse/SCRUM-2649
    .map(simplifySpeciesNameSC)
    .filter((value) => (
      value && value.trim()
    ));
};
```

Reply:
And if there is a reference in the UI and we should change it.

Reply:
Ok... this method is only used in the Integration tests in the Java software. I propose that we change its name from `get` to retrieve then we don't need the `@JsonIgnore`